cmd/main.go

```diff
@@ -85,6 +85,8 @@ var (
 	// Serve live profiles of the process (CPU, memory, etc.) over HTTP on this port
 	httpProfilePort   = flag.Int("httpprof_port", 0, "Port to serve HTTP profiles from")
 	foldRemoteRootSvg = flag.Bool("fold_remote_root_svg", false, "Whether to fold root SVG from remote mixer")
+	// Cache SV -> inputPropertyExpressions for StatisticalCalculations
+	cacheSVFormula = flag.Bool("cache_sv_formula", true, "Whether to cache SV -> inputPropertyExpresions for StatisticalCaclulations.")
```

Comment:
can we make it default to be false? Then set it true in deployment. After PR merge, can test custom DC and make sure it works.

Add a TODO tot change this back to "true" after this is fully released?

Reply:
Good idea, done